### PR TITLE
URA-570 eggd conductor uranus mye config v2.1.0

### PR DESCRIPTION
**Pull request by Aisha-D**
- Replace eggd_vcf_handler_for_uranus inputs with inputs for eggd_vep, eggd_vcf_rescue and egg_generate_varaint_workbook
- Update workflow ID
- Update readme to reflect new input files with the new apps

- - -
This change is [Reviewable](https://reviewable.io/reviews/eastgenomics/eggd_conductor_configs/89)

## Summary by CodeRabbit

- **New Features**
	- Updated the myeloid assay configuration to version 2.1.0, introducing new workflow components and input parameters.
	- Added new applications for `eggd_vep` and `eggd_vcf_rescue` with specified input fields and files.
- **Bug Fixes**
	- Removed outdated references to the `eggd_vcf_handler_for_uranus` application, streamlining input file organisation.
- **Documentation**
	- Enhanced clarity in the readme file regarding input file specifications and versions, including updates to the `panel_bed` fields.

### CODE REVIEW

File: assay_configs/MYE/readme.md [hunk 1]
```diff
@@ -29,27 +29,14 @@ Below defines all external input files, their versions and file locations in DNA
...
+||`panel_bed`|[coding_unrestricted_athena_GRCh38_myeloid_5bp_flank_v2.2.0.bed](https://platform.dnanexus.com/projects/Fkb6Gkj433GVVvj73J7x8KbV/data/bed_files/b38/kits/myeloid?id.values=file-Gqy3qk847xBjBk1G7YV6YJ9V)|v2.2.0|
+|[eggd_vep [cgppindel VCF]](https://github.com/eastgenomics/eggd_vep)|`config_file`|[uranus_vep_config_v1.0.1.json](https://platform.dnanexus.com/projects/Fkb6Gkj433GVVvj73J7x8KbV/data/dynamic_files/vep_configs?id.values=file-Gx47xzQ42ffzPPXyZqQ9bjK7)|v1.0.1|
+||`panel_bed`|[pindel_cgppindel_filtering_coordinates_v1.2.bed](https://platform.dnanexus.com/projects/Fkb6Gkj433GVVvj73J7x8KbV/data/bed_files/b38/kits/myeloid?id.values=file-Gqy4V1Q47xBXYzfKxb22fGXz)|v1.2|
+|[eggd_vcf_rescue](https://github.com/eastgenomics/eggd_vcf_rescue)|`rescue_vcf`|[CosmicCodingMuts_GRCh38_v99.normal.NoEntry.vcf.gz](https://platform.dnanexus.com/projects/Fkb6Gkj433GVVvj73J7x8KbV/data/?id.values=file-GjFv9X0421zQFGxXVFQ4j972)|v99|
+||`fasta_tar`|[GRCh38_GIABv3_no_alt_analysis_set_maskedGRC_decoys_MAP2K3_KMT2C_KCNJ18_noChr.fasta-index.tar.gz](https://platform.dnanexus.com/projects/Fkb6Gkj433GVVvj73J7x8KbV/data/?id.values=file-Gb772VQ4XGyzQ0Zk615XbZ0X)| not versioned |
```
[the remark this answers is not in the document]
Comment (Aisha-D) [reply assay_configs/MYE/readme.md:37]: Genome reference files dont have versioning in the same manner as annotation sources, hence they are not versioned

File: assay_configs/MYE/readme.md [hunk 2]
```diff
@@ -29,27 +29,14 @@ Below defines all external input files, their versions and file locations in DNA
...
+
+
```
[the remark this answers is not in the document]
Comment (Aisha-D) [reply assay_configs/MYE/readme.md]: done

File: assay_configs/MYE/eggd_conductor_uranus_MYE_config_v2.1.0.json [hunk 3]
```diff
@@ -82,84 +83,44 @@
...
+                "stage-eggd_generate_variant_workbook.summary": "uranus",
+                "stage-eggd_generate_variant_workbook.split_hgvs": true,
+                "stage-eggd_generate_variant_workbook.af_format": "percent",
+                "stage-eggd_generate_variant_workbook.report_text": true,
+                "stage-eggd_generate_variant_workbook.join_columns": "Prev_Count=CSQ_Prev_Count_AC,/,CSQ_Prev_Count_NS",
+                "stage-eggd_generate_variant_workbook.keep_filtered": true,
+                "stage-eggd_generate_variant_workbook.keep_tmp": false,
+                "stage-eggd_generate_variant_workbook.add_comment_column": false,
+                "stage-eggd_generate_variant_workbook.add_classification_column": false,
+                "stage-eggd_generate_variant_workbook.merge_vcfs": false,
```
[the remark this answers is not in the document]
Comment (Aisha-D) [reply assay_configs/MYE/eggd_conductor_uranus_MYE_config_v2.1.0.json:123]: new lines breaks the code so better to have it extra long

File: assay_configs/MYE/eggd_conductor_uranus_MYE_config_v2.1.0.json [hunk 4]
```diff
@@ -82,84 +83,44 @@
...
+                "stage-eggd_vcf_rescue.rescue_vcf": {
+                    "$dnanexus_link": "file-GjFv9X0421zQFGxXVFQ4j972"
                 },
-                "stage-eggd_vcf_handler_for_uranus.maf_file_tbi": {
-                    "$dnanexus_link": "file-Gj6YQg84qv8Z3F4J8fX7YK6V"
+                "stage-eggd_vcf_rescue.fasta_tar": {
+                    "$dnanexus_link": "file-Gb772VQ4XGyzQ0Zk615XbZ0X"
                 },
+                "stage-eggd_vcf_rescue.rescue_filtered": true,
+                "stage-eggd_vcf_rescue.filter_string": "bcftools filter -e '(FORMAT/DP<99 | AF<0.03 | CSQ_gnomADe_AF>0.1) & (CSQ_RESCUE_LIST!=\"RESCUED\")'",
```
[the remark this answers is not in the document]
Comment (Aisha-D) [reply assay_configs/MYE/eggd_conductor_uranus_MYE_config_v2.1.0.json]: Done.

**Review by Aisha-D — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/eastgenomics/eggd_conductor_configs/89)* status: 0 of 2 files reviewed, 5 unresolved discussions (waiting on @cnascim1 and @coderabbitai[bot])

**Review by cnascim1 — APPROVED** [review]
> Reviewed 1 of 1 files at r3, 1 of 1 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/eastgenomics/eggd_conductor_configs/89)* status: all files reviewed, 5 unresolved discussions (waiting on @Aisha-D)

**Pull request merged**